{"name": "测试用例描述：错误的验证码登录失败", "status": "passed", "description": "测试描述：错误的验证码登录失败", "steps": [{"name": "输入正确账号", "status": "passed", "attachments": [{"name": "正确账号", "source": "55fdac6d-6cfd-431d-a5cf-423168d93ef8-attachment.attach"}], "start": 1651654666572, "stop": 1651654666572}, {"name": "输入正确密码", "status": "passed", "attachments": [{"name": "正确密码", "source": "2d3a223d-4c45-42e3-be12-c97d36dfe895-attachment.attach"}], "start": 1651654666607, "stop": 1651654666607}, {"name": "输入错误验证码", "status": "passed", "attachments": [{"name": "验证码为空", "source": "0a766232-eebe-4fd4-a7e3-01b8fe9791c9-attachment.attach"}], "start": 1651654666653, "stop": 1651654666653}], "attachments": [{"name": "截图", "source": "511af2f7-a470-4373-9c3c-f0a11bbc2bd5-attachment.png", "type": "image/png"}], "start": 1651654666572, "stop": 1651654669173, "uuid": "41942f57-c152-4333-8e5f-f6d6f720ddc2", "historyId": "99cc6b199ab7eac5979ecaecd144eda1", "testCaseId": "44097febf60fbfed56a361cecfc05a9e", "fullName": "test_two.Test#test_07", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "severity", "value": "critical"}, {"name": "epic", "value": "xxx测试"}, {"name": "story", "value": "登录"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "6940-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}